app/grandchallenge/algorithms/models.py

```diff
@@ -97,7 +97,7 @@ class Algorithm(UUIDModel, TitleSlugDescriptionModel):
     additional_terms_markdown = models.TextField(
         blank=True,
         help_text=(
-            "By using this algortihm, users agree to the site wide "
+            "By using this algorithm, users agree to the site wide "
```

Comment:
If you change anything on a model field you also need to regenerate the migrations. In the case of a help text change this is one of the few that do not change the DB schema, if you're confident of that (test it) you can also correct the typo in the last migration where this field was updated. But, adding a new migration is the easiest!

Reply:
Ah, just saw a typo. Will create a migration.